practice-project-api/.env

```diff
@@ -0,0 +1 @@
+PORT=8000
```

Comment:
I should have mentioned this but put `.env` in `.gitignore`. This is because different people will want different `.env` files. This is more of a local dev thing